lib/samples/add_feature_layer_with_time_offset/add_feature_layer_with_time_offset.dart

```diff
@@ -187,7 +187,7 @@ class _AddFeatureLayerWithTimeOffsetState
 
     // The URL of the feature layer showing hurricanes.
     final featureLayerUri = Uri.parse(
-        'https://sampleserver6.arcgisonline.com/arcgis/rest/services/Hurricanes/MapServer/0');
+        'https://sampleserver6.arcgisonline.com/arcgis/rest/services/Hurricanes/MapServer/0',);
```

Comment:
With the addition of the new commas in all the files, the code should end up getting reformatted a little, the next time anyone does `dart format`. The automated `dart fix` stuff doesn't do that for us, so we should run `dart format` on the sample repo. Could you run `dart format` in the samples repo? That'll clean up a bunch of stuff. (Unfortunately, it'll also clean up some stuff caused by the recent "remove Samples from the class name" change, but that should be fine.)